tdrs-backend/tdpservice/users/test/factories.py

```diff
@@ -22,9 +22,31 @@ class Meta:
     last_name = factory.Faker("last_name")
     is_active = True
     is_staff = False
-    stt = factory.SubFactory(STTFactory)
 
     @classmethod
     def _create(cls, model_class, *args, **kwargs):
         manager = cls._get_manager(model_class)
         return manager.create_user(*args, **kwargs)
+
+    @factory.post_generation
+    def groups(self, create, extracted, **kwargs):
+        """Add groups to user instance."""
+        if not create:
+            return
+
+        if extracted:
+            for group in extracted:
+                self.groups.add(group)
+
+class UserFactory(BaseUserFactory):
+    """General purpose user factory used through out most tests."""
+
+    stt = factory.SubFactory(STTFactory)
+
+class STTUserFactory(BaseUserFactory):
+    """User factory for use in STT tests."""
+    # To prevent an error that happens when calling the `populate_stt` command.
+    # The stt factory and the command were competing over the right to set the stt.
+    # Our solution was to not set the STT specificly for the STT tests that
```

Comment:
```suggestion
    # Our solution was to not set the STT specifically for the STT tests that
```

Reply:
@carltonsmith @riatzukiza did you see this one? how should these sorts of fixes be made going forward?